docs/src/templates/pages/styleguide/ui-components/custom/star-ratings.hbs

```diff
@@ -9,6 +9,16 @@ navactive: star-ratings
 layout: components
 ---
 
+<h2 class="sg-sectionHeading">Extra small rated stars</h2>
```

Comment:
Just a heads up that this will need updating now, as it changed to be `c-rating--small`